### PR TITLE
Fix next(), print(), and raw_input() to work in Python 2 and 3

### PR DESCRIPTION
**Pull request by cclauss**
Related to #4552

Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

For an explanation of how to fill out the fields, please see the relevant section 
in [PULL_REQUESTS.md](./PULL_REQUESTS.md)

*Description*: Resolves syntax errors and name errors that have the potential to halt the runtime.
*Risk Level*: Minimal
*Testing*: http://flake8.pycqa.org
*Docs Changes*:
*Release Notes*:
[Optional Fixes #Issue]
[Optional *Deprecated*:]

### CODE REVIEW

File: tools/check_format_test_helper.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 # docker. Normally this is run via check_format_test.sh, which
 # executes it in under docker.
 
+from __future__ import print_function
```
Comment (htuch) [inline tools/check_format_test_helper.py]: Can you be consistent in putting a separating blank line between the `__future__` imports and the rest? I think the space makes sense, but it's not done everywhere.

File: tools/deprecate_version/deprecate_version.py [hunk 2]
```diff
@@ -31,6 +33,11 @@
 import github
 from git import Repo
 
+try:
+  input = raw_input  # Python 2
```
Comment (htuch) [inline tools/deprecate_version/deprecate_version.py]: I think the scope of this PR is more than just `print`, it seems general Python 3 compat, can you update the description?

File: tools/check_format_test_helper.py (hunk 1 above)
Comment (cclauss) [reply tools/check_format_test_helper.py]: Fixed.

File: tools/deprecate_version/deprecate_version.py (hunk 2 above)
Comment (cclauss) [reply tools/deprecate_version/deprecate_version.py]: Fixed.

**Pull request merged**